### PR TITLE
[enhancement](pipeline) support jdbc scan, jdbc sink, odbc scan and odbc sink node for pipeline.

### PR DESCRIPTION
**Pull request by luozenglin**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.45 seconds
 load time: 462 seconds
 storage size: 17123356802 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221210122845_clickbench_pr_61358.html

**Review by Gabriel39 — APPROVED** [review]
LGTM

File: be/src/pipeline/pipeline_fragment_context.cpp [hunk 1]
```diff
@@ -300,7 +300,9 @@ Status PipelineFragmentContext::_build_pipelines(ExecNode* node, PipelinePtr cur
     auto node_type = node->type();
     switch (node_type) {
     // for source
-    case TPlanNodeType::OLAP_SCAN_NODE: {
```
Comment (HappenLee) [inline be/src/pipeline/pipeline_fragment_context.cpp:303]: seems mysql scan node can be same here ？

Comment (luozenglin) [reply be/src/pipeline/pipeline_fragment_context.cpp:303]: The current `VMysqlScanNode` inherits from `ScanNode`, and later can unify support for `NewMysqlScanNode` inherits from `VScanNode`

**Pull request merged**